src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -763,7 +763,7 @@ private void OnChildAdded(UIElement child)
 				}
 				else
 				{
-					child.ManagedOnLoaded(Depth.Value + 1);
+					child.ManagedOnLoaded(Depth.GetValueOrDefault(int.MinValue) + 1);
```

Comment:
```suggestion
					child.ManagedOnLoaded((Depth ?? int.MinValue) + 1);
```
Would be better as it avoids invoking a generic method.